eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -36,6 +36,11 @@ parameters:
   type: boolean
   default: false
 
+- name: useDevVersions
+  displayName: True when build output uses dev/CI versioning instead of official build versioning
+  type: boolean
+  default: false
```

Comment:
is this supposed to be just for testing in the PR? we could also just pass `/p:UseOfficialBuildVersioning=false` through the existing `extraProperties`

Reply:
As we already expose this as a new build switch, we should use it here and not pass the msbuild property in. That said, given that this is temporary, I wonder if it's even worth introducing the --dev switch in the scripts.

Reply:
Given that we're pausing the Unified Build initiative for a bit, I'd rather have a well-documented switch for this important scenario until we decide to flip the default experience to be non-official-versioned instead of a hard-to-remember MSBuild property.